File: NetFramework/InputPanel/VB/BillOfSale/SaleInfo.vb
Public Class SaleInfo
    Public Property DateSold As DateTime
    Public Property Seller As Transactor
    Public Property Buyer As Transactor
    Public Property ItemsInfo As String
    Public Property ItemsValue As Integer
    Public Property FormOfPayment As String

    Public Sub New(ByVal dateS As DateTime, ByVal s As Transactor, ByVal b As Transactor, ByVal itemsInfo As String, ByVal itemsValue As Integer, ByVal formPayment As String)
        Me.DateSold = dateS
        Me.Seller = s
        Me.Buyer = b
        Me.ItemsInfo = itemsInfo
        Me.ItemsValue = itemsValue
        Me.FormOfPayment = formPayment
    End Sub
End Class

Public Structure Transactor
    Public Property PrintedName As String
    Public Property Address As String
    Public Property City As String
    Public Property State As String
    Public Property Zip As String
    Public Property HomePhone As String
    Public Property CellPhone As String
    Public Property Work As String
    Public Property DateSign As DateTime

    Public Sub New(ByVal printName As String, ByVal adress As String, ByVal city As String, ByVal state As String, ByVal zip As String, ByVal homePhone As String, ByVal cellPhone As String, ByVal work As String, ByVal dateSign As DateTime)
        Me.PrintedName = printName
        Me.Address = adress
        Me.City = city
        Me.State = state
        Me.Zip = zip
        Me.HomePhone = homePhone
        Me.CellPhone = cellPhone
        Me.Work = work
        Me.DateSign = dateSign
    End Sub
End Structure

File: NetFramework/InputPanel/CS/BillOfSale/SaleInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BillOfSale
{
    class SaleInfo
    {
        public DateTime DateSold { get; set; }
        public Transactor Seller { get; set; }
        public Transactor Buyer { get; set; }
        public string ItemsInfo { get; set; }
        public int ItemsValue { get; set; }
        public string FormOfPayment { get; set; }
        public SaleInfo(DateTime dateS, Transactor s, Transactor b, string itemsInfo, int itemsValue, string formPayment)
        {
            DateSold = dateS;
            Seller = s;
            Buyer = b;
            ItemsInfo = itemsInfo;
            ItemsValue = itemsValue;
            FormOfPayment = formPayment;
        }

    }
    struct Transactor
    {
        public string PrintedName { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Zip { get; set; }
        public string HomePhone { get; set; }
        public string CellPhone { get; set; }
        public string Work { get; set; }
        public DateTime DateSign { get; set; }

        public Transactor(string printName, string adress, string city, string state, string zip,
            string homePhone, string cellPhone, string work, DateTime dateSign)
        {

            PrintedName = printName;
            Address = adress;
            City = city;
            State = state;
            Zip = zip;
            HomePhone = homePhone;
            CellPhone = cellPhone;
            Work = work;
            DateSign = dateSign;
        }

    }
}
